jaksel-interpreter.js

```diff
@@ -132,6 +135,34 @@ const consoleLog = (msg) => {
   }
 }
 
+const incrementVar = (msg) => {
+  let format = /([a-zA-Z0-9]+) healthy$/
+  let match = msg.match(format)
+  if (!match) return null;
+
+  return {
+    exp: `${match[1]}++;`
+  }
+}
+
+const decrementVar = (msg) => {
+  let format = /([a-zA-Z0-9]+) toxic$/
+  let match = msg.match(format)
+  if (!match) return null;
+  return {
+    exp: `${match[1]}--;`
+  }
+}
+
+const commentLine = (msg) => {
+  let format = /IMO (.*)/
```

Comment:
I think kayaknya lebih bagus kalo this dibuat alternative with "FYI" juga, jadinya bisa

FYI ini tuh comment